course-mw/sunbird-util/sunbird-platform-core/common-util/src/main/java/org/sunbird/common/util/CloudStorageUtil.java

```diff
@@ -51,26 +51,34 @@ public static String getSignedUrl(
             Some.apply("r"), Some.apply("application/pdf"));
   }
 
-  private static BaseStorageService getStorageService(String storageType) {
-    String storageKey = PropertiesCache.getInstance().getProperty(JsonKey.ACCOUNT_NAME);
-    String storageSecret = PropertiesCache.getInstance().getProperty(JsonKey.ACCOUNT_KEY);
-    return getStorageService(storageType, storageKey, storageSecret);
-  }
+	public static String getAnalyticsSignedUrl(String storageType, String container, String objectKey) {
```

Comment:
Is this method required.

Reply:
this method (getAnalyticsSignedUrl) was there in the release tag release-5.0.0_RC2. I dont see this in the release-5.0.2 branch. Is it obsolete now

Reply:
removed obsolete method getAnalyticsSignedUrl